Sources/NumericsShims/include/NumericsShims.h

```diff
@@ -143,6 +143,18 @@ HEADER_SHIM float libm_log10f(float x) {
   return __builtin_log10f(x);
 }
 
+HEADER_SHIM float libm_ceilf(float x) {
```

Comment:
These don't appear to be used anywhere?

Reply:
No, I only use the `double` version in the BigInt implementation, but I thought for the sake of completeness I would add all the variants. If that's wrong, I'd be happy to take 'em out!

Reply:
As noted at the use sites, you don't need any of these.